language/move-lang/functional-tests/tests/designated_dealer/burns.move

```diff
@@ -40,5 +40,9 @@ script {
     }
 }
 
-// check: MintEvent
+// check: ReceivedMintEvent
```

Comment:
Could you keep the check for `MintEvent` as well (since both events should be emitted)?